storage/object/api-cli/combining-iam-and-object-storage.mdx

```diff
@@ -1,280 +1,174 @@
+    ```json
+    {
+    "name": "S3-policy",
+    "description": "Grant full access to Object Storage to S3-group in S3-Project",
+    "organization_id": "example-orga-id-81df-11532f025b28",
+    "rules": [
         {
-            "Sid": "allow all actions",
-            "Effect": "Allow",
-            "Principal": {
-                "SCW": "application_id:950dde46-5cba-427d-a4f5-ce5a8a79717c"
-            },
-            "Action": [
-                "*"
-            ],
-            "Resource": [
-                "production-bucket-1",
-                "production-bucket-1/*"
-            ]
+        "permission_set_names": [
+            "ObjectStorageFullAccess"
+        ],
+        "project_ids": [
+            "S3-project-id-4c04-9e03-f48147eee1f1"
+        ]
         }
-    ]
-}
-```
+    ],
+    "group_id": "S3-group-id-b86e-1d960b4c54cd"
+    }
+    ```
+3. [Create a bucket policy](/storage/object/api-cli/create-bucket-policy/) to manage access to bucket A.
+    ```json
+    {
+    "Version": "2023-06-07-alpha",
+        "Id": "bucket-A-policy",
+        "Statement": [
+            {
+                "Sid": "Allow user to GET and PUT objects",
+                "Effect": "Deny",
+                "Principal": "user-id-1d96-0b4c-54cd",
+                "NotAction": [
+                    "s3:GetObject",
+                    "s3:PutObject"
+                ],
+                "Resource": [
+                    "bucket-A",
+                    "bucket-A/*"
+                ]
+            }
+        ],
+        "Statement": [
+            {
+                "Sid": "Allow application to GET objects",
+                "Effect": "Deny",
+                "Principal": "application-id-1d96-0b4c-54cd",
+                "NotAction": [
+                    "s3:GetObject",
+                ],
+                "Resource": [
+                    "bucket-A",
+                    "bucket-A/*"
+                ]
+            }
+        ]
+    }
+    ```
+4. [Create a bucket policy](/storage/object/api-cli/create-bucket-policy/) to manage access to bucket B.
+    ```json
+    {
+    "Version": "2023-06-07-alpha",
+        "Id": "bucket-B-policy",
+        "Statement": [
+            {
+                "Sid": "Deny access to all except application",
+                "Effect": "Deny",
+                "NotPrincipal": "application-id-1d96-0b4c-54cd",
+                "Action":  "s3:*",
+                "Resource": [
+                    "bucket-B",
+                    "bucket-B/*"
+                ]
+            }
+        ],
+        "Statement": [
+            {
+                "Sid": "Allow application to PUT and DELETE objects",
+                "Effect": "Deny",
+                "Principal": "application-id-1d96-0b4c-54cd",
+                "NotAction": [
+                    "s3:PutObject",
+                    "s3:DeleteObject",
```

Comment:
Out of curiosity, we have two possible actions for deleting objects?

Reply:
Delete exactly 1 object and delete up to 1,000 objects